java/maven/src/org/netbeans/modules/maven/api/output/OutputUtils.java

```diff
@@ -241,7 +244,8 @@ public ProjectStacktraceOutputListener(Project project) {
         protected ClassPath getClassPath() {
             Project prj = ref.get();
             if(prj != null) {
-                ClassPath[] cp = prj.getLookup().lookup(ProjectSourcesClassPathProvider.class).getProjectClassPaths(ClassPath.EXECUTE);
+                ProjectSourcesClassPathProvider provider = prj.getLookup().lookup(ProjectSourcesClassPathProvider.class);
+                ClassPath[] cp = provider.getProjectClassPaths(ClassPath.BOOT);
```

Comment:
Are you sure that still works for anything _not_ in the JDK?  I would expect to have `ClassPath.BOOT` added to `ClassPath.EXECUTE` in there?

Reply:
first version i tested indeed merged both but the application is already included in the boot cp.

going to step through the lazy eval lambda hell of ClassPathProviderImpl later and check if this is just a bug or by design. Maven deps is another case i have to check

Reply:
it doesn't contain maven deps. Going to revert to the original version.